examples/application/build.gradle

```diff
@@ -0,0 +1,62 @@
+import proguard.gradle.ProGuardTask
+import org.gradle.internal.jvm.Jvm
+
+buildscript {
+    repositories {
+        mavenCentral()
+        google()
+    }
+    dependencies {
+        classpath 'com.guardsquare:proguard-gradle:7.2.1'
+    }
+}
+
+plugins {
+    id 'application'
+    id 'com.guardsquare.proguard' apply false
```

Comment:
```suggestion
```

This isn't required for Java projects, since there's no ProGuard plugin for Java projects only the ProGuardTask.

Reply:
How should I change the sample?

Reply:
This line should just be deleted, as is the sample doesn't build ;)